### ISSUE TITLE
Generate full bibliography in Obsidian

### ISSUE THREAD

**Issue by nickelswitte** [mgmeyers/obsidian-zotero-integration#175]
Hello,
I was trying to find information on this, but I did not find any.

I would like to cite several authors from my Zotero database and then generate one full bibliography, like with Zotero in Word. Currently, when I use the plugin, I am only able to get one entry of a full bibiography, but not the whole one, of all references used.

![grafik](https://user-images.githubusercontent.com/36443026/215901908-3f4c5cfd-8092-4817-b14b-c399afe248d0.png)

Basically, I am just looking for the same funcitonality which Microsoft Word also offers.

Thanks for the help!

**Comment by rkshthrmsh** [comment]
Following. I am also looking for something like this.

**Comment by nickelswitte** [comment]
I found out that Zettlr, am open-source alternative to Obsidian is able to do this (see [here](https://docs.zettlr.com/en/academic/citations/)). But I would want not to switch Obsidian to Zettlr for this.

**Comment by TheFermi0n** [comment]
I wish to achieve a similar level of functionality. Presently, the bibliography is inserted at the same location where command is called, but I would love to get a system similar to Microsoft Word + Zotero's bibliography integration (auto update at the bottom of the document).

**Comment by psychologistcy** [comment]
> I wish to achieve a similar level of functionality. Presently, the bibliography is inserted at the same location where command is called, but I would love to get a system similar to Microsoft Word + Zotero's bibliography integration (auto update at the bottom of the document).

I wish, too.
